source/elements/oneMKL/source/domains/dft/enums_and_config_params.rst

```diff
@@ -262,6 +265,16 @@ the :ref:`descriptor<onemkl_dft_descriptor>` class.
             :math:`M > 1`.
         -   | ``std::int64_t``
             | [0]
+      * -   :ref:`WORKSPACE_PLACEMENT<onemkl_dft_config_workspace_placement>`
+        -   Some FFT algorithm computation steps require a scratch space for permutations or other purposes. 
+            This parameter controls whether this scratch space is automatically allocated or provided by the user.
+        -   | :ref:`onemkl_dft_enum_config_value` (possible values are ``config_value::WORKSPACE_AUTOMATIC`` or ``config_value::WORKSPACE_EXTERNAL``).
+            | [``config_value::WORKSPACE_AUTOMATIC``]
+      * -   WORKSPACE_EXTERNAL_BYTES
+        -   The required minimum external workspace size for use by :ref:`set_workspace<onemkl_dft_descriptor_set_workspace>`.
+            A read-only value.
+        -   | ``std::int64_t``
```

Comment:
This should throw if called on an uncommitted descriptor.

Reply:
Addressed in [8d71156](https://github.com/oneapi-src/oneAPI-spec/pull/509/commits/8d71156cdb56d1645593168c20de1137e4fd18e5).